### PR TITLE
add welcome dialog on first boot and more.

### PR DESCRIPTION
**Pull request by Zangetsu38**
- Add welcome dialog with some information for first boot and new user.
-  gui/app context menu: fix glitch on render.
- emu: bump on version 0.1.2

# Result:
image

### CODE REVIEW

File: vita3k/CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 set(VITA3K_ORG_NAME "Vita3K")
 set(VITA3K_APP_NAME "Vita3K")
-set(VITA3K_APP_VERSION "v0.1")
+set(VITA3K_APP_VERSION "v0.0.2")
```
Comment (1whatleytay) [inline vita3k/CMakeLists.txt:3]: 🤔 We're going back a few versions?

Comment (Zangetsu38) [reply vita3k/CMakeLists.txt:3]: i just is to early for can bump on 0.2, so i have thinks using like rpcs3? for can bump  version more quickely, seriously i have set this 0.1 in 2018 hmhm lol

File: vita3k/gui/src/welcome_dialog.cpp [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    ImGui::Separator();
+    ImGui::Spacing();
+    ImGui::TextColored(GUI_COLOR_TEXT, "Vita3k is an open-source PlayStation Vita emulator written in C++ for Windows, Linux and MacOS.");
+    ImGui::TextColored(GUI_COLOR_TEXT, "\nThe emulator is still in it's early stages so any feedback and testing is greatly appreciated.");
+    ImGui::TextColored(GUI_COLOR_TEXT, "\nTo get started, please install the Vita firmware and font packages.");
+    ImGui::Spacing();
+    ImGui::Spacing();
+    ImGui::TextColored(GUI_COLOR_TEXT, "A comprehensive guide on how to set-up Vita3k can be found on the");
+    ImGui::SameLine();
+    std::ostringstream link;
```
Comment (IllusionMan1212) [inline vita3k/gui/src/welcome_dialog.cpp:53]: if you could please put this at the start of the function, it's kinda lost between all the gui code here

**Comment by IllusionMan1212** [comment]
if you could also align the buttons with the text

File: vita3k/gui/src/welcome_dialog.cpp (hunk 2 above)
Comment (Zangetsu38) [reply vita3k/gui/src/welcome_dialog.cpp:53]: I see

**Comment by Zangetsu38** [comment]
> if you could also align the buttons with the text

Is little complicated usint g sale line get this

**Review by IllusionMan1212 — DISMISSED** [review]
looks good. please wait until CI finishes building before merging

**Review by IllusionMan1212 — APPROVED** [review]

**Pull request merged**